### ISSUE TITLE
Cannot setup the demo?

### ISSUE THREAD

**Issue by YingshanDeng** [ckeditor/ckeditor5-hackathon#10]
Hello, I follow the README installation step but cannot setup the demo. When I visit http://localhost:1030/build/amd/tests/hackathon/manual/  It only show 
![image](https://cloud.githubusercontent.com/assets/7246595/16177374/4cb19596-365d-11e6-8bde-ae8115eb7b62.png)
And when I run the test, it show 
![image](https://cloud.githubusercontent.com/assets/7246595/16177380/7f689a84-365d-11e6-9b19-e06cc78c62de.png)

Can you give me some help? Thank u very much!😘

**Comment by Reinmar** [comment]
Hey, great to see that the demos interested you :).

Unfortunately, CKEditor 5 is under heavy development, so I even added a note to the [installation steps](https://github.com/ckeditor/ckeditor5-hackathon#installation) that the code will stop working at some point, and apparently that moment already happened. We renamed few packages recently and since there's no versioning yet (we're planning to start tagging in couple of weeks), it'd be hard to get back to the state from 2 weeks before.

However, I generated all the builds while they were still working so I think I can add them to the repository. I'll let you know once I do that.
